dotcom-rendering/makefile

```diff
@@ -152,9 +152,6 @@ bundlesize: clear clean-dist install build
 validate: clean-dist install tsc lint stylelint test validate-build
 	$(call log, "everything seems 👌")
 
-validate-prepush:
```

Comment:
Can I just check why this is being removed here? It looks like this make command was doing other things in addition to running `prettier`. Is it the case that we don't need the command at all?

Reply:
i coudln't find any reference to it in the code base https://github.com/search?q=repo%3Aguardian%2Fdotcom-rendering+validate-prepush&type=code so don't think it's actually in use?

Reply:
I thought that might be it, but I wasn't sure because I don't think `validate-ci` is referenced elsewhere either? So I thought perhaps they were there for manual use. But in that case I agree that it seems reasonable to remove it and see if anyone actually notices? Thanks!